walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java

```diff
@@ -19,6 +19,7 @@
 import java.util.HashSet;
 import java.util.Set;
 
+
```

Comment:
Exactly one blank line should follow the import statements: https://google.github.io/styleguide/javaguide.html#s3-source-file-structure